src/sparseml/transformers/finetune/data/custom.py

```diff
@@ -66,3 +71,14 @@ def get_raw_dataset(self, *_ignore, **__ignore) -> DatasetDict:
             )
 
         return raw_dataset
+
+    def get_remove_columns_from_dataset(self, dataset) -> List[str]:
+        """Remove redandant columns from the dataset for processing"""
+        remove_columns = set()
+        for key in dataset.keys():
```

Comment:
Nit: you could use dataset.values() here since you don't use key for anything else